packages/itwin/ec3-widget/src/EC3Widget.ts

```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import type { Localization, TranslationOptions } from "@itwin/core-common";
+import { IModelApp } from "@itwin/core-frontend";
+import type { EC3LocalizationResult } from "./components/EC3/EC3Localization";
+
+export interface EC3WidgetConfig {
+  overRiddenStrings?: EC3LocalizationResult;
+}
+
+/** EC3Widget is use when the package is used as a dependency to another app.
+ * '''ts
+ *  await EC3Widget.initialize({...EC3WidgetConfigProps});
+ * '''
+ * @beta
+ */
+export class EC3Widget {
+  private static _defaultNs = "ec3Widget";
+  public static localization: Localization;
+  private static overridenStrings?: EC3LocalizationResult;
+
+  /** Used to initialize the EC3Widget */
+  public static async initialize(config?: EC3WidgetConfig): Promise<void> {
+    // register namespace containing localized strings for this package
+    EC3Widget.localization = IModelApp.localization;
```

Comment:
Add a way to provide different localization. Might be useful for tests.

Example: https://github.com/iTwin/viewer-components-react/blob/master/packages/itwin/reports-config-widget/src/ReportsConfigWidget.ts#L24C3-L24C79

Reply:
done